<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园通知推送客户端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>分类筛选：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryCombo">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>全部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>行政</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学术</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>生活</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>艺术</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>体育</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="refreshBtn">
        <property name="text">
         <string>同步最新通知</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { padding: 5px 15px; font-weight: bold; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="text">
         <string>增加通知</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { padding: 5px 15px; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editBtn">
        <property name="text">
         <string>修改通知</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { padding: 5px 15px; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteBtn">
        <property name="text">
         <string>删除通知</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { padding: 5px 15px; }</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="buttonSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
     </widget>
    </item>
    
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>就绪</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray; font-size: 11px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>